### PR TITLE
mkdocs.yml: Add syntax highlighting

### PR DESCRIPTION
**Pull request by ayan-b**
| Before | After |
|--------|-------|
|image|image|

### CODE REVIEW

**Comment by sanketplus** [comment]
thanks @ayan-b for the PR. does this require any external package? in that case `requirements.txt` needs to be updated

**Comment by ayan-b** [comment]
@sanketplus, no, it doesn't require any external package as [the extensions are already installed](https://github.com/squidfunk/mkdocs-material/blob/master/requirements.txt#L25) along with `mkdocs-material`.

**Comment by sanketplus** [comment]
@ayan-b would you also be willing to tag the language to the code blocks in the repo if missing?
ref: https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#specifying-the-language

**Comment by ayan-b** [comment]
@sanketplus Done!

File: courses/security/fundamentals.md [hunk 1]
```diff
@@ -42,7 +42,7 @@
   - Fail securely
     - Applications regularly fail to process transactions for many reasons. How they fail can determine if an application is secure or not.
 
-    ```
+    ```java
```
Comment (sanketplus) [inline courses/security/fundamentals.md:45]: @stormic-nomad-nishant can you confirm this? otherwise LGTM

Comment (stormic-nomad-nishant) [reply courses/security/fundamentals.md:45]: LGTM

**Pull request merged**